ncm-metaconfig/src/main/metaconfig/httpd/pan/schema.pan

```diff
@@ -386,15 +387,33 @@ type httpd_perl_handler = {
     "responsehandler" : string
 };
 
-type httpd_wsgi_iportscript = {
+type httpd_wsgi_importscript = {
     "path" : string
     "process" ? string
     "application" ? string
 };
 
+type httpd_wsgi_daemonprocess = {
+    "name" : string
+    @{converted in list of key=value}
```

Comment:
panc on Jenkins is falling over the equals in this annotation, time to update to 10.7?

Reply:
sure, but then there's no way back.you can move the milestone to a release where we can expect that 10.7 is ok to run. this is a parse error, not a compilation error, so afaik, as soon as this temnplate in your repo, you must be running 10.7

Reply:
Hmm, could you remove the `=` then?